doc/source/examples/resolved-cfd-dem/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -0,0 +1,345 @@
+	* update the RHS
+	* solve the matrix system
+	* reevaluate the RHS to check the convergence.
+	
+	
+Linear Solver Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+	subsection linear solver
+	  set method                                 = gmres
+	  set max iters                              = 1000
+	  set max krylov vectors 		     = 1000
+	  set relative residual                      = 1e-4
+	  set minimum residual                       = 1e-11
+	  set ilu preconditioner fill                = 0
+	  set ilu preconditioner absolute tolerance  = 1e-20
+	  set ilu preconditioner relative tolerance  = 1.00
+	  set verbosity                              = verbose
+	end
+	
+* The ``method`` is set to ``gmres``. This solver is cheaper than the other option, and this case doesn't require any special preconditioner. This makes the ``gmres`` solver the best option available.
+
+* The ``max iters`` is set to 1000. This is a lot more steps than how much it should take to solve the system.
+
+* The ``max krylov vectors`` is set to 1000. This is to ensure that we keep the full Arnoldi basis for each new iteration. From experience keeping a maximum of Krylov vector results in a faster resolution for this case than clearing the basis after a certain number of ``gmres`` iterations.
+
+* The ``relative residual`` is set to 1e-4. This is small enough, so we don't under-resolve our matrix and do extra non-linear steps because of it, and at the same, it doesn't require too many ``gmres`` iterations.
+
+* The ``ilu preconditioner fill`` is set to 0. This means that we don't have any preconditioner. This is the cheapest option. In this case, we are able to use this option without having to do too many ``gmres`` iterations. It requires less computational time to do a few more  ``gmres`` iterations than building the preconditioner and doing fewer ``gmres`` iterations.
+
+IB particles
+~~~~~~~~~~~~~~
+.. code-block:: text
+
+	subsection particles
+		set number of particles = 1
+		set stencil order =3
+		set refine mesh inside radius factor = 0.8
+		set refine mesh outside radius factor = 1.3
+		set initial refinement = 6
+		set fluid density =0.000960
+		set integrate motion = true
+		set assemble Navier-Stokes inside particles = false
+		set length ratio = 2
+		set particle nonlinear tolerance=1e-5
+		set alpha =1
+		subsection gravity
+		    	set Function expression =0;-981;0
+		end
+		subsection particle info 0 
+			subsection position
+			    set Function expression =5;12.75;5
+			end
+			subsection velocity
+			    set Function expression =0;0;0
+			end
+			set radius = 0.75
+			set density= 0.001120
+	    	end
+
+	end
+
+This section is where we define most of the parameters that are related to the particle.
+
+
+* The ``number of particles`` is set to one as we only want one particle.
+
+* The ``stencil order`` is set to 3 as this is the highest order we can use for this case, and it won't lead to Runge instability.
```

Comment:
```suggestion
* The ``stencil order`` is set to 3 as this is the highest order we can use for this case, and it will not lead to Runge instability.
```